cmd/downloads/config/downloads_config_test.go

```
@@ -213,7 +215,7 @@ func TestHandler_BlocksUntilArchivesReady(t *testing.T) {
 	t.Run("archive request blocks then succeeds", func(t *testing.T) {
 		done := make(chan int, 1)
 		go func() {
-			req := httptest.NewRequest(http.MethodGet, "/amd64/linux/oc.tar", nil)
+			req := httptest.NewRequest(http.MethodGet, "/amd64/linux/oc.tar.gz", nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)